recipes/jupyter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: jupyter
+  version: {{ version }}
+
+source:
+  fn: jupyter-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/j/jupyter/jupyter-{{ version }}.tar.gz
+  sha256: d9dc4b3318f310e34c82951ea5d6683f67bed7def4b259fafbfe4f1beb1d8e5f
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Can this be `pip install --no-deps .` so that appropriate metadata files are written?